examples/actix3_example/api/src/lib.rs

```diff
@@ -109,7 +109,7 @@ async fn edit(data: web::Data<AppState>, id: web::Path<i32>) -> Result<HttpRespo
     let post: post::Model = Query::find_post_by_id(conn, id)
         .await
         .expect("could not find post")
-        .unwrap_or_else(|| panic!("could not find post with id {}", id));
+        .unwrap_or_else(|| panic!("could not find post with id {id}"));
```

Comment:
These must belong to the "cleanup clippy" PR

Reply:
Right.

Reply:
https://github.com/SeaQL/sea-orm/pull/1436